content/release-10-15-0/applications-10-15-0-bundle/10-15-0-152.md

```diff
@@ -0,0 +1,78 @@
+---
+weight: 36
+title: Release 10.15.0.152
+layout: bundle
+---
+
+<!--10.15.0.115 - 10.15.0.152-->
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>
+Web SDK</td>
+<td>A set of new <code>ApplicationOptions</code> (<code>mapWidgetPageSize</code>, <code>mapWidgetRealtimeDisabled</code>, <code>mapWidgetHideMaxDeviceOnMapHint</code>) specific to the Map widget have been introduced. These allow, for example, to increase the default page size of the Map widget. For details, see <a href="http://resources.cumulocity.com/documentation/websdk/1013.0.292/ngx-components/classes/ApplicationOptions.html#mapWidgetHideMaxDeviceOnMapHint" class="no-ajaxy">http://resources.cumulocity.com/documentation/websdk/1013.0.292/ngx-components/classes/ApplicationOptions.html#mapWidgetHideMaxDeviceOnMapHint</a>.
+<td>MTM-48427</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>
+Cockpit</td>
+<td>Instead of an alert, a warning message is now shown when deleting assets in the <b>Subassets</b> page, which informs the user that the total number shown in the assets counter might be incorrect.</td>
+<td>
+MTM-47676</td>
+</tr>
+
+<tr>
+<td>
+Web SDK</td>
+<td>The initial requests of the map widget have been reduced. The location update events are only retrieved on demand for a specific device instead of requests for all devices.</td>
+<td>
+MTM-48251</td>
+</tr>
+
+<tr>
+<td>
+Web SDK</td>
+<td>A issue was resolved, that let the <code>c8y-messages</code> component throw a "Parameter "key" required" error when used in combination with a <code>c8y-form-group</code>.</td>

```

Comment:
```suggestion
<td>An issue was resolved, that let the <code>c8y-messages</code> component throw a "Parameter "key" required" error when used in combination with a <code>c8y-form-group</code>.</td>
```
there's also double " quotes. as this is text, it's probably not an issue, but it would have to be avoided in code.